### ISSUE TITLE
Update via ScriptDude

### ISSUE THREAD

**Issue by tonmar77** [marcusraitner/COVID-19-Dashboard#38]
Ich update das Skript per ScriptDude. Kann es sein das die Version im Gist nicht immer die aktuellste ist?

**Comment by marcusraitner** [comment]
Ja. Das ist aber auch Absicht. Die Idee ist, dass ich dort nur Releases aktiv pushe.

**Comment by tonmar77** [comment]
Alles klar, danke.

**Comment by marcusraitner** [comment]
Vielleicht sollte ich mir aber überlegen, hier einen Development-Branch zu machen anstatt wie jetzt trunk-based zu entwickeln.
